cmd/kops/create_cluster.go

```diff
@@ -170,6 +172,7 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringVar(&options.Image, "image", options.Image, "Image to use")
 
 	cmd.Flags().StringVar(&options.Networking, "networking", "kubenet", "Networking mode to use.  kubenet (default), classic, external, kopeio-vxlan, weave, flannel, calico, canal.")
+	cmd.Flags().BoolVar(&options.NetworkEncrypt, "network-encrypt", options.NetworkEncrypt, "Weave network support encryption between nodes. Password stored in etcd as k8s secret weave-pass.")
```

Comment:
Let me attempt to be a bit more clear.  We do not support all configurations with flags.  For more advanced usage, which this is, we want to have a user to use YAML to define those options.  See: `kops create -f`.

Here is the how I export the yaml to work with clusters:

```
$KOPS get cluster $1 -o yaml > $1.yaml
echo " " >> $1.yaml
echo "---" >> $1.yaml
echo " " >> $1.yaml
$KOPS get ig --name $1 -o yaml >> $1.yaml
```
You can now access the weave options that you have added to the api: https://github.com/kubernetes/kops/pull/2276/files#diff-4044ed3abd7d6925f80ed8661dab94e1L56

kops has a functionality for storing secrets already. ssh pem keys, admin passwords, etc.  Please look at the code for secrets and reuse that code for the weave secret.  For instance https://github.com/kubernetes/kops/blob/master/cmd/kops/create_secret_sshpublickey.go